Sources/SwiftFCSDKiOS/SwiftFCSDKiOS.docc/Symbols/ACBClientCall.md

```diff
@@ -18,11 +18,43 @@ Please see <doc:VideoCalls> for an explanation of how to use ACBClientCall
     @objc final public var remoteView: UIView?
 
     /// The call's ``UIView`` used for displaying remote buffer views intended for use in picture in picture enabled applications
+    @available(*, deprecated, message: "use sampleBufferView() async -> UIView? instead, This method will be removed in future versions of FCSDKiOS")
     @objc final public func remoteBufferView() async -> UIView?
 
+    /// The call's ``UIView`` used for displaying remote buffer views intended for use in picture in picture enabled applications
+    @MainActor @objc final public func sampleBufferView() async -> UIView?
+
     /// The client should always remove the `remoteBufferView()` when finished using the bufferView.
+    @available(*, deprecated, message: "use removeSampleView() async instead, This method will be removed in future versions of FCSDKiOS")
     @objc final public func removeBufferView() async
 
+    /// The client should always remove the `remoteBufferView()` when finished using the bufferView.
+    @objc final public func removeSampleView() async
```

Comment:
Just thinking about the name of these functions - I know the scope of this review is this documentation, but sampleBufferView makes less sense than remoteBufferView to me - could it be something like remoteCameraView and removeRemoteCamerView that are used instead? But if the SDK is already out with the names given, this shouldn't be changed.

Reply:
Also it might be worth looking into what we call these in other versions of the SDK, e.g. android and web.